db/data/test-data/test-data.sql

```diff
@@ -1,35 +1,35 @@
-DROP DATABASE IF EXISTS nc_games_test;
+-- DROP DATABASE IF EXISTS nc_games_test;
```

Comment:
im not sure what this file is, but ideally i'd try to only commit files pertinent to the ticket.